liquidprompt

```diff
@@ -2343,6 +2348,28 @@ _lp_runtime_color() {
     lp_runtime_color="${LP_COLOR_RUNTIME}${lp_runtime_format}${NO_COL}"
 }
 
+######################
+# Kubernetes context #
+######################
+
+_lp_kube_context() {
+   (( LP_ENABLE_KUBECONTEXT )) || return
+
+   local _LP_KUBE_CONTEXT=$(kubectl config current-context 2>/dev/null)
+   if [[ "$LP_DELIMITER_KUBECONTEXT" ]]; then
+     lp_kube_context="${_LP_KUBE_CONTEXT%%${LP_DELIMITER_KUBECONTEXT}*}"
+   else
+     lp_kube_context="$_LP_KUBE_CONTEXT"
+   fi
```

Comment:
You can drop the else, because this needs to be passed through `__lp_escape()`:
```suggestion
   fi
 
    local ret
    __lp_escape "$kubernetes_context"
    lp_kube_context=$ret
```